locales/en.yml

```
@@ -0,0 +1,200 @@
+no_favorites: "No favorites yet."
+no_tracks_found: "No tracks found."
+unknown_artist: "Unknown Artist"
+unknown_album: "Unknown Album"
+unknown_title: "Unknown Title"
+unknown_genre: "Unknown"
+most_played_local_tracks: "Your most played local tracks."
+no_tracks_in_library: "No tracks found in your library."
+no_songs_here: "No songs here."
+syncing_with_server: "Syncing with server..."
+most_played_tracks: "Your most played tracks."
+no_more_songs: "No more songs in queue"
```

YAMLlint (1.38.0)

[error] duplication of key "no_more_songs" in mapping

(key-duplicates)